### PR TITLE
Add climbing assist selector

### LINKED ISSUE #69: Add Climb Assist Checkbox(es?)
We should know if the robot can assist other robots in climbing. We should be able to track whether it can lift two robots at the same time, also.

### PR DESCRIPTION
**Pull request by AndrewLester**
Fixes #69 

I felt like a drop down would be better than two checkboxes (where at least one will not be used most of the time)

### CODE REVIEW

**Comment by ErikBoesen** [comment]
This is fine. Do you think it might be better, though, if we had two separate select boxes: Did it (not attempt/attempt/succeed) to lift a first robot, and (not attempt/attempt/succeed) to lift a second one? Let me know what you think about this.

**Comment by AndrewLester** [comment]
I see what you mean, actually. The attempt item would remind us to ask them whether their robot normally lifts another.

File: index.html [hunk 1]
```diff
@@ -67,6 +67,22 @@ <h2>Climbed?</h2>
...
+            <select id="assisted-first">
+                <option>No Attempt</option>
+                <option>Attempt</option>
+                <option>Succeed</option>
+            </select>
+        </div>
+        <div>
+            <h2>Assisted Second Robot?</h2>
+            <select id="assisted-second">
+                <option>No Attempt</option>
```
Comment (ErikBoesen) [inline index.html]: Just a tip: `<select>`s get reset to `No`, so this will cause issues. I'll fix this myself though.

**Comment by ErikBoesen** [comment]
@AndrewLester I mainly want us to be able to track how many assists a robot is capable of.

**Pull request merged**